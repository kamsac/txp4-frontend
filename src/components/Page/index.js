import React from 'react';
import Navbar from '../Navbar';

export default class Page extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="Page">
        <Navbar/>
        <div className="PageContent">
          {this.props.children}
        </div>
      </div>
    );
  }
}
