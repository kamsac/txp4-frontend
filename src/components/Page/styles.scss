@import '../../variables';

.PageContent {
  grid-area: PageContent;
}
