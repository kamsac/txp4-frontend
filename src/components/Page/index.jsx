import React from 'react';
import PropTypes from 'prop-types';
import './styles.scss';
import NavbarContainer from '../../containers/NavbarContainer';

export default class Page extends React.Component {
  render() {
    return (
      <div className="Page">
        <NavbarContainer />
        <div className="PageContent">
          {this.props.children}
        </div>
      </div>
    );
  }
}

Page.propTypes = {
  children: PropTypes.element.isRequired,
};
