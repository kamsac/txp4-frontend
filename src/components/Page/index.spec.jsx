import React from 'react';
import { shallow } from 'enzyme';
import Page from './index';

test('<Page>', () => {
  const component = shallow(<Page><div>Something</div></Page>);
  expect(component).toMatchSnapshot();
});
