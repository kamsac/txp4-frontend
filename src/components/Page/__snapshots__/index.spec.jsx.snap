// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page> 1`] = `
<div
  className="Page"
>
  <Navbar />
  <div
    className="PageContent"
  >
    <div>
      Something
    </div>
  </div>
</div>
`;
